hypertrace-view-generator/helm/templates/view-generation/view-gen-individual-config.yaml

```diff
@@ -0,0 +1,14 @@
+{{ $global := . }}
+{{- range $k, $val := .Values.viewGenIndividualConfigs }}
```

Comment:
Is it possible to use same iterator - `$group.generator.serviceConfig.viewgenerators` ?

Reply:
That iterator is for groups, this iterator is for individual view-gens. Groups are independant from individual config.